Create 20170912-prediction-of-huawei-sales-of-vidhance-models.md

vvizard- opened the pull request:
Proposed article about Huawei sales predictions

vvizard- committed:
Create 20170912-prediction-of-huawei-sales-of-vidhance-models.md

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+author: Mr IMINT
+---
+
+# Prediction of Huawei sales of Vidhance models
```
You don't have to give the title as an header. It is added automatically. This also means that you can reduce header levels by one in the rest of the document.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+
+# Prediction of Huawei sales of Vidhance models
+
+## Disclaimer
```
I suggest calling it _Background_ instead. It sounds a little less boring.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+| **Honor 9** | 2017 Jun | **>1M** [(1)][H9_1MONTH] | - | - | - | - | - | - | - | - | - | - | - |
+
+### Mate 9 Family
+The Mate 9 family includes the "Mate 9", "Mate 9 Pro" and "Mate 9 Porsche design". The "Mate 9 Lite" is not included, as it does not run Vidhance software. The Mate 9 Family was announced in October 2016, and went on sale in China in November. Availability in most of western Europa, along with the US was not in place until January 2017.
```
Maybe state very early in the article that Lite are totally different models and then define the Mate 9 Series and use that everywhere.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+
+The only data we got for the P10 so far, is that it had sold above 6 million units in it's 3rd month. The P9 didn't reach 6 million until it's 5th month. So the P10 seem to have outsold the P9 by a __huge margin__ for it's first quarter. Unfortunately that's all the data we got for the P10 so far, but I expect to get an update on this at end of 2017 (or before), as it certainly looks on track to  ...
+
+___Huaweis previous flagship record is held by the P9 which sold 12 million units within it's first 11 months. The only datapoint we got for P10 so far, shows it's selling 36% better than the P9 did at that point in time. As Huawei have seen a overall 20.6% increase in 2017H1 on total unit shipments (all models), and P10 seemingly a very big success with the consumers, I'd say a _modest_ 25% incr ...
```
Second last word should be "hasn't".

simonmika replied:
Single underscore again.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+___Huaweis previous flagship record is held by the P9 which sold 12 million units within it's first 11 months. The only datapoint we got for P10 so far, shows it's selling 36% better than the P9 did at that point in time. As Huawei have seen a overall 20.6% increase in 2017H1 on total unit shipments (all models), and P10 seemingly a very big success with the consumers, I'd say a _modest_ 25% incr ...
+
+### Honor 9
+This is tough. Detailed sales about any Honor model are hard to come by. We know that yearly sales of the Honor subsidiary accounted for approximately 60 million units of Huawei's total 139.3 shipped units.  Other than that, the only data available is that Honor 8 sold 1.5 million units by it's second month after release, while the Honor 9 (with Vidhance) have exceeded 1 million during it's first ...
```
"while the Honor 9 (with Vidhance) _has_ exceeed"

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+### Honor 9
+This is tough. Detailed sales about any Honor model are hard to come by. We know that yearly sales of the Honor subsidiary accounted for approximately 60 million units of Huawei's total 139.3 shipped units.  Other than that, the only data available is that Honor 8 sold 1.5 million units by it's second month after release, while the Honor 9 (with Vidhance) have exceeded 1 million during it's first ...
+
+__Based on strong year-on-year growth for the Honor brand, and the "Honor 9" realtively cheap chinese price, compared to the P10, one could expect this to sell a lot of units. But it got a lot of competition from other Huawei / Honor models. Earlier this year, Honor released the "Honor V9" in china, which was rebranded into the "Honor 8 Pro" in international markets. It's specifications and price ...
```
Try only to highlight a few words at a time. These long paragraphs are not suitable to skim through. Instead try to find the few words in every part that actually highlights the conclusion.

simonmika reviewed (commented):
Good in general. Maybe it might be worth splitting it into one piece talking about which models contain Vidhance and another about the potential volumes. The second can reference the first. Just a suggestion. Great work by the way.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+
+Looking at all data we got for all relevant Huawei/Honor phones, monthly unit shipping seems to be very strong (actually near it's peak) around the 7th month after a release. It should be safe to assume Mate 9 is still selling in relative good volumes. 
+
+___Based on similarity with P9 in sales volumes so far, the fact that Mate-series is usually outsold by the P-series, but accounting for that 2017 so far have shown a 20.6% unit shipment increase from Huawei, I'm guessing the Mate 9 will sell somwhere in the ballpark of the P9 in it's first 12 months. More specifically 10-12 million units.___
```
Use single underscore.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+* P10 family
+* Honor 9
+
+> __I expect these three models to combined sell between 33 to 38 million units during their first 12 months__.
```
Don't use underscore for the full quote. And make the quote as self explanatory as possible while still being short enough for Twitter.

simonmika replied:
And move it higher up in the document (just a few lines) and leave a version of it here as well.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+ * 2016: 139.3 million units shipped (+30.2% year-on-year)  [(source: Huawei)](http://consumer.huawei.com/en/press/news/2017/hw-u-035103/)
+ * 2017H1: 73.01 million units shipped (+20.6% year-on-year) [(source: Huawei)](http://www.huawei.com/za/news/za/huawei-consumer-business-group-announces-2017-h1-business-results)
+ 
+> _"By the third quarter of 2016, 60% of Huawei’s smartphone shipments were mid- and high-priced devices, a striking reversal of its past reliance on cheap handsets."_
```
Don't add quote signs when using `>` as they will be added by the css. Also remember that quotes a flown outside of the regular text so you might too have a shorter version in the quote and a longer version left here. Also don't emphasize full sentences.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+My claims regarding Vidhance beeing part of the P10 have been peer reviewed by others, and my claim of Vidhance beeing used in the Honor 9 is a peer review of another persons original claim. There are several community members who have run analysis on these firmwares, and come to the same conclusions. For the remainder of this article, Vidhance beeing used in Mate 9, P10 and Honor 9 is considered ...
+
+Both the Mate 9 and the P10 comes in several variants. The variants are
+|Name| Running Vidhance stabilization?|
```
You need another newline before the table for it too work.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+
+My claims regarding Vidhance beeing part of the P10 have been peer reviewed by others, and my claim of Vidhance beeing used in the Honor 9 is a peer review of another persons original claim. There are several community members who have run analysis on these firmwares, and come to the same conclusions. For the remainder of this article, Vidhance beeing used in Mate 9, P10 and Honor 9 is considered ...
+
+Both the Mate 9 and the P10 comes in several variants. The variants are
```
End line with `:`.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+### General sales
+Here is a table trying to give every known sales volumes by offical (or at least credible) sources, for all Huawei and Honor models that might be relevant for predicting the sales volumes of the models running Vidhance. _Italic data points_ are estimations based upon other __known datapoints, marked in bold__. Paranthesis mark the source of information for the datapoint.
+
+| Model | Released | Month 1 | Month 2 | Month 3  | Month 4  | Month 5  | Month 6  | Month 7  | Month 8  | Month 9  | Month 10  | Month 11  | Month 12|
```
This table is too wide for the page. I might find a way to fix it but we should consider rearranging the data somehow.

simonmika commented on content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md:
```diff
@@ -0,0 +1,135 @@
+
+That is counting each model from it's release, and 12 months out. They are likely to continue selling some millions (combined) even after 12 months, but a lack of good datapoints at this make it hard to predict from after 12 months.
+
+---
```
This line makes no sense as the links below actually will show up at their usage.

simonmika reviewed (changes requested):
I will have to update the css to handle the images.

vvizard- committed:
Update 20170912-prediction-of-huawei-sales-of-vidhance-models.md

Fixed error in a table

vvizard- committed:
Update 20170912-prediction-of-huawei-sales-of-vidhance-models.md

Fixed errors, implemented suggestions.

vvizard- committed:
Update 20170912-prediction-of-huawei-sales-of-vidhance-models.md

Fixed a layout error in the last conclusion.

vvizard- committed:
Update 20170912-prediction-of-huawei-sales-of-vidhance-models.md

Few more fixes. Don't have time to split the article into seperate pieces at the moment unfortunately, but I agree, it makes sense to keep them seperate.

simonmika committed:
Simplified some formatting.

Stick to only one type emphasizing with single underscore.

vvizard- commented:
Regarding CSS and images. What was the problem? Is it something you will update, or should I make changes? If so, what?

simonmika commented:
I already fixed the images.
